queries/polygonal_change.sql

```diff
@@ -7,138 +7,136 @@ These queries identify polygonal features that have changed from 2011 to 2019. T
 */
 
 -- Find polygons from 2011 that are different in 2019
-CREATE TABLE results_diff_geo.polygonal_2011_change AS
+CREATE TABLE results_final.polygonal_2011_change AS
 SELECT
 	*
-FROM mn_2011_upload.multipolygons
+FROM mn_2011_upload.multipolygons_new
 WHERE  NOT EXISTS (
    SELECT
-   FROM results_diff_geo.polygonal_no_change_all r
-   WHERE  osm_id = r.osm_id_t1 OR osm_way_id = r.osm_way_id_t1
+   FROM results_final.polygonal_no_change r
+   WHERE  osm_id_all = r.osm_id_t1
    )
 ;
 
 -- Find polygons from 2019 that are different from 2011
-CREATE TABLE results_diff_geo.polygonal_2019_change AS
+CREATE TABLE results_final.polygonal_2019_change AS
 SELECT
 	*
-FROM mn_2019_upload.multipolygons
+FROM mn_2019_upload.multipolygons_new
 WHERE  NOT EXISTS (
    SELECT
-   FROM results_diff_geo.polygonal_no_change_all r
-   WHERE  osm_id = r.osm_id_t1 OR osm_way_id = r.osm_way_id_t1
+   FROM results_final.polygonal_no_change r
+   WHERE  osm_id_all = r.osm_id_t1
    )
 ;
 
 -- Subset the change results from 2011 for only NHD features
-CREATE TABLE results_diff_geo.polygonal_2011_change_nhd AS
+CREATE TABLE results_final.polygonal_2011_change_nhd AS
 SELECT
 	*
-FROM results_diff_geo.polygonal_2011_change
+FROM results_final.polygonal_2011_change
 WHERE
 	LOWER(source) LIKE '%nhd%'
 ;
 
 -- Subset the change results for 2019 for only NHD features
-CREATE TABLE results_diff_geo.polygonal_2019_change_nhd AS
+CREATE TABLE results_final.polygonal_2019_change_nhd AS
 SELECT
 	*
-FROM results_diff_geo.polygonal_2019_change
+FROM results_final.polygonal_2019_change
 WHERE
 	LOWER(source) LIKE '%nhd%'
 ;
 
 -- Subset the change results from 2011 for features where only the geometry changed
-CREATE TABLE results_diff_geo.linear_2011_change_nhd_geom AS
+CREATE TABLE results_final.polygonal_2011_change_nhd_geom AS
```

Comment:
did you mean to change this from linear to polygonal?

Reply:
@alxswny Yeah...must have copied in linear stuff into the polygonal file by accident